udata_hydra/analysis/csv.py

```diff
@@ -65,6 +67,28 @@
 RESERVED_COLS = ("__id", "tableoid", "xmin", "cmin", "xmax", "cmax", "ctid")
 
 
+async def notify_udata(check_id):
+    """Notify udata of the result of a parsing"""
+    check = await get_check(check_id)
+    resource_id = check["resource_id"]
+    db = await context.pool()
+    record = await db.fetchrow("SELECT dataset_id FROM catalog WHERE resource_id = $1", resource_id)
+    if record:
+        payload = {
+            "resource_id": resource_id,
+            "dataset_id": record["dataset_id"],
+            "document": {
+                "analysis:parsing:table": check["parsing_table"],
```

Comment:
@geoffreyaldebert should we keep `parsing:table` since we're using resource_id as the pivot value for tabular api?

Reply:
Upon discussion, we can remove `parsing:table` :)